test/PromissoryNote.ts

```diff
@@ -235,7 +239,7 @@ describe("PromissoryNote", () => {
             await expect(PromissoryNote.connect(signers[0]).initialize(signers[0].address)).to.not.be.reverted;
 
             await expect(PromissoryNote.connect(signers[1]).setPaused(true)).to.be.revertedWith(
-                "AccessControl: account 0xadd93e738a415c5248f7cb044fcfc71d86b18572 is missing role 0xa49807205ce4d355092ef5a8a18f56e8913cf4a201fbe287825b095693c21775",
+                `AccessControl: account ${signers[1].address.toLowerCase()} is missing role ${"0xa49807205ce4d355092ef5a8a18f56e8913cf4a201fbe287825b095693c21775"}`,
```

Comment:
I think these should be fixed in the other PR where you fixed `LoanCore.ts`, for a couple reasons:

1) We are not going to merge roku's suggested changes before submitting a new commit hash for QS (to avoid expanded scope). And we want these fixes included in the new commit hash
2) Easier to give a single commit pointing to "we fixed tests here" as opposed to two